File: redsheet-portal/src/app/database-models/v-email-template.ts
import { required, maxLength, range, nested } from '@rx/annotations';

export class vEmailTemplate {
    constructor(vEmailTemplate?: vEmailTemplate )  {
        let properties = [ "emailTemplateId", "statusId", "subject",];
        for (let property of properties)
            if (vEmailTemplate && vEmailTemplate[property])
                this[property] = vEmailTemplate[property];
    }
 
	emailTemplateId : number =   0 ;
 
    @range(1,2147483647)
	statusId : number =   undefined;
 
    @required()
    @maxLength(200)
	subject : string =   undefined;


}

File: RedSheet.DbEntities/Models/vEmailTemplate.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vEmailTemplates", Schema = "dbo")]
	[Serializable]
	public partial class vEmailTemplate  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.Identity)] 
	[Key]
	public int EmailTemplateId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int StatusId { get; set; } 
 
	[Required] 
	[MaxLength(200)]
	public string Subject { get; set; } 

	public vEmailTemplate()
	{
    }

    }
}

